.pylintrc

```diff
@@ -1,7 +1,49 @@
 [MASTER]
-disable=missing-function-docstring,
+# This is a workaround for pylint not being able to trace our changing of the
+# import path in tests so we can find the actual module.
+init-hook='import sys; sys.path.append(".")'
```

Comment:
I'm not fully grasping why the tests do that. One reason might be that it allows to run tests after cd'ing into `tests/`.

But that's unusual I think and to me seems okay to remove the `sys.path` wrangling in the tests and document to use top-level commands instead as that will add `.` to the PYTHONPATH iiuc.
```
python3 -m unittest discover -s ./tests
python3 -m unittest   ./tests/test_brokertypes.py
pytest
```
The websocket.py import as is looks a bit tricky (though it seems to work even if the sys.path inserts are removed). I suspect that could be solved differently by "mocking" actual classes on the real `websocket` module rather than the import hijacking and stub out approach.

Reply:
Interesting. I added this while trying to address stuff added to the allowlist with this patch. For what is in this patch it is probably not needed though and was more intended to make the life of whoever works on cleaning this up simpler. I can drop it here until we need it.

Reply:
Yeah, not having the `init-hook` in the config would be my preference as that hopefully should not be needed.